src/App.jsx

```diff
@@ -1,18 +1,12 @@
-import React from 'react'
-import { Routes, Route, BrowserRouter, Link } from 'react-router-dom'
-import {
-  HomePage,
-  NotFoundPage,
-  DetailsPage,
-  BookmarksPage
-} from './pages'
+import React from "react";
+import { Routes, Route, BrowserRouter, Link } from "react-router-dom";
+import { HomePage, NotFoundPage, DetailsPage, BookmarksPage } from "./pages";
 
-import { Header } from './components'
+import { Header } from "./components";
 
 function App() {
-
   return (
-    <>
+    <div className="container px-8">
```

Comment:
@robfuncken could you please remove this container? Micha asked to remove the px-8 as well. 
That's what he said!!